config/.gitignore

```diff
@@ -1,6 +1,5 @@
-# Ignore everything in this directory
-*
```

Comment:
The `config` directory contains end-user settings files, thus it need to have this ignore rule in place